packages/remix/lib/components/progress/progress.dart

```diff
@@ -29,11 +30,13 @@ base class ProgressSpec extends Spec<ProgressSpec>
     BoxSpec? container,
     BoxSpec? track,
     BoxSpec? fill,
+    BoxSpec? outer,
     super.animated,
     super.modifiers,
   })  : container = container ?? const BoxSpec(),
         track = track ?? const BoxSpec(),
-        fill = fill ?? const BoxSpec();
+        fill = fill ?? const BoxSpec(),
+        outer = outer ?? const BoxSpec();
```

Comment:
I think we should name it outerContainer, or use a more descriptive name if available.

Reply:
done